auth-web/package.json

```diff
@@ -20,7 +20,7 @@
     "@bcrs-shared-components/corp-type-module": "1.0.13",
     "@bcrs-shared-components/enums": "1.1.7",
     "@bcrs-shared-components/interfaces": "1.0.71",
-    "@mdi/font": "^4.5.95",
+    "@mdi/font": "^5.9.55",
```

Comment:
This could have some regressions, it's a major version

Reply:
I'd recommend getting the list of the MDI icons or fonts used, including the one in the database and checking they all exist in this new version, but that isn't so easy to do I think. Or read the changelog? see if any were removed?

Reply:
Probably will be fine, just ask Ashna to look at all of the icons